packages/terra-clinical-slide-group/src/SlideGroup.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import Slide from './Slide';
+
+import './SlideGroup.scss';
+
+const propTypes = {
+  /**
+   * The array of components for the group. Only the last component is visible. The others are hidden but still mounted.
+   */
+  items: PropTypes.array,
+  /**
+   * When true, the transition between slides is not animated.
+   */
+  animationIsDisabled: PropTypes.bool,
+};
+
+const SlideGroup = ({ items, animationIsDisabled, ...customProps }) => {
+  // We don't want to render the transition group when no children exist. Doing so will cause the first child to
+  // animate into place, which in most cases we do not want.
+  if (!items || !items.length) {
+    return null;
+  }
+
+  const slideGroupClassNames = classNames([
+    'terraClinical-SlideGroup',
+    customProps.className,
+  ]);
+
+  // We use the key from the first child as the key for the transition group. This will cause the transition group to
+  // rerender when the root child changes and subsequently prevent that child from animating into position.
+  const transitionGroupKey = items[0].key;
+
+  const itemCount = items.length;
+
+  return (
+    <div {...customProps} className={slideGroupClassNames}>
+      <CSSTransitionGroup
+        key={transitionGroupKey}
+        transitionEnter={!animationIsDisabled}
```

Comment:
Check out my comment above. I do not think it would be weird to default animations to be on for the slide group. In all other components we have created, a user that wants to disable a particular boolean must explicitly define `booleanName={false}` (or true if the other way around).